style-file.css

```diff
@@ -0,0 +1,75 @@
+html{
+    background-color: #ffdd99;
+}
+
+.header-style{
+    width: 100%;
+    color: #ffdd99;
+    background-color: #336600;
+    text-align: center;
+}
+
+.header-style h1{
+    padding: 10px;
+    font-size: 50px;
+}
+
+.nav-style{
+    
+}
+
+.nav-style ul{
+    background-color: #336600;
+    text-align: center;
+    margin: 0px;
+    padding: 0px;
+    overflow: hidden;
+}
+
+.nav-style ul li{
+    display: inline;
+    text-align: center;
+    font-size: 30px;
```

Comment:
better use em instead of px for fonts

see https://kyleschaeffer.com/css-font-size-em-vs-px-vs-pt-vs-percent